cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -284,11 +293,12 @@ def render_init_scripts(api_servers):
     kubelet_opts = FlagManager('kubelet')
     # Declare to kubelet it needs to read from kubeconfig
     kubelet_opts.add('--require-kubeconfig', None)
-    kubelet_opts.add('--kubeconfig', '/srv/kubernetes/config')
+    kubelet_opts.add('--kubeconfig', kubeconfig_path)
+    kubelet_opts.add('--network-plugin', 'cni')
```

Comment:
I think it'd be doable, but not trivial. If SDN and CNI are different interfaces, then we'd have to consider some weird cases - for example, what happens if we end up with relations to both?

For what it's worth, Flannel has a CNI plugin so my preference would be to drop SDN entirely and see if we can get Flannel working through CNI instead.